flybywiresim/a32nx
# feat: w&b integration with advanced Simbrief API

Fixes #[issue_no]

## Summary of Changes

Makes PAX_WEIGHT and BAG_WEIGHT dynamic to support the updated Simbrief API's ``<weights>`` section.
We are now able to pull separated weight values that can be customised on individual Simbrief airframes as per pilot/company requirements. 
Default values are set to 84KGs (Pax weight+Cabin luggage) and 20KGs (check-in baggage per pax). Option to customize the default weight values without using Simbrief would be planned for the EFB load management interface.

## Screenshots (if necessary)


![image](https://user-images.githubusercontent.com/29005005/153646302-7eca887e-82da-4616-8374-1efddf65f52b.png)
![image](https://user-images.githubusercontent.com/29005005/153646419-2084cc95-0f80-4d93-a378-afd1c3aded02.png)

## References





## Additional context



Discord username (if different from GitHub): sidnov#8337

## Testing instructions

1. Spawn C/D
2. Prepare a Simbrief Flightplan with the new airframe or use your own custom airframe.
3. Launch devmode->behaviors->local variables and filter using "WB"
4. Observe that your pax and bag weight from Simbrief is being loaded correctly on pressing OFP REQUEST
5. Input pax numbers manually to see the weights change to default (84,20)
6. Start boarding and observe the payload value after completion. Cross check with Simbrief OFP values. 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
